.gitignore

```diff
@@ -1,3 +1,4 @@
+/composer.lock
```

Comment:
Thanks. I am aware what it does. However, if you install concise as a dependency into a project, the lock files are ignored and the dependencies are resolved by Composer, taking into account platform, required dependencies by other dependencies, et cetera.

This means that running CI against a specific set of dependencies will give you a false impression of how well it will work as a dependency for _other_ projects. 

So you should test against the latest versions of dependencies and update your project's `composer.json` to reject versions of dependencies for which concise is not compatible. If you use a lock file, you'll never find out issues.